src/plugins/credential_management/public/types.ts

```diff
@@ -3,14 +3,41 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import {
+  ChromeStart,
+  ApplicationStart,
+  IUiSettingsClient,
+  OverlayStart,
+  SavedObjectsStart,
+  NotificationsStart,
+  DocLinksStart,
+  HttpSetup,
+} from 'src/core/public';
+import { DataPublicPluginStart } from 'src/plugins/data/public';
 import { NavigationPublicPluginStart } from '../../navigation/public';
+import { ManagementAppMountParams } from '../../management/public';
 
-export interface CredentialManagementPluginSetup {
-  getGreeting: () => string;
-}
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface CredentialManagementPluginStart {}
+import { CredentialManagementStart } from './index';
+import { OpenSearchDashboardsReactContextValue } from '../../opensearch_dashboards_react/public';
 
 export interface AppPluginStartDependencies {
   navigation: NavigationPublicPluginStart;
```

Comment:
Did you want to remove this since it's not a required plugin anymore? could also add the datapublicpluginstart here too.

Reply:
not sure if we really need data plugin here, I suppose not

Reply:
Removed data plugin and related logic